src/freenas/usr/local/bin/freenas-debug

```diff
@@ -274,6 +274,10 @@ main()
 	if $all_debug; then
 		func_pids=""
 		for opt in ${opts_spaced} ; do
+			if [ ${opt} == 'B' ]
```

Comment:
That can cause issues if "opt" is a white-space. Also '=' is what you use in shell :)

if [ "${opt}" = "B" ]